src/pad.ts

```diff
@@ -2,24 +2,18 @@
  *   Notepad Widget
  */
 
-/** @module UI.pad
+/** @module pad
  */
+import store from './store'
+import ns from './ns'
```

Comment:
widgets, store, ns are missing type declarations.. just a note

Reply:
Mm, will be added as we clean them up.